src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -2929,6 +2929,21 @@ public void testIndentationLineBreakVariableDeclaration()
         verifyWarns(checkConfig, fileName, expected);
     }
 
+    @Test
+    public void testLambdaInEnum() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+        checkConfig.addProperty("tabWidth", "4");
```

Comment:
Update this configuration to be consistent with issue report.

Reply:
`tabWidth` property?
or you mean to change `forceStrictCondition` to true?

Reply:
Please see issue report. User is using the default `tabWidth`. It is always a good idea to use the exact configuration provided in bug reports to avoid inconsistencies.

Reply:
Got it.
Updated pls check

Reply:
Resolved.